packages/marko/docs/webpack.md

```diff
@@ -76,17 +76,20 @@ export default [
     plugins: [markoPlugin.server],
   },
   {
-    rules: [
-      {
-        test: /\.marko$/,
-        loader: "@marko/webpack/loader",
-      },
-      // If using `style` blocks with Marko you must use an appropriate loader
-      {
-        test: /\.css$/,
-        use: ["style-loader", "css-loader"],
-      },
-    ],
+    module: {
+      entry: "./browser.js",
```

Comment:
I don't know if this is right. But omitting it leads to the error:

```
TypeError: Cannot read properties of undefined (reading 'markoEntriesPending')
```

Reply:
The marko server plugin should populate the entry for the browser, so if that isn't working then this would be a bug.